Cargo.toml

```diff
@@ -3,6 +3,12 @@
 inherits = "test"
 strip = "debuginfo"
 
+[profile.release-with-debug]
+inherits = "release"
+debug = true
+lto = true
+codegen-units = 1
+
```

Comment:
Where does this get used? Any idea of the perf differences without these settings? I imagine this should become the default if it does make an impact?

Reply:
No, I was just using this for testing -- can be removed.